src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Error.txt

```diff
@@ -117,3 +117,7 @@ Errors: Error 0-10: 'Wyz' is an unknown or unsupported function.
 
 >> Error(Table({Kind:ErrorKind.Div0},{Kind:ErrorKind.Numeric}))
 Error(Table({Kind:ErrorKind.Div0},{Kind:ErrorKind.Numeric}))
+
+// No duplicated errors
+>> 1.2.3
```

Comment:
> 1.2.3 [](http://example.com/codeflow?start=2&length=6)

Can we test expression which has same error message but at different locations?
e.g. `1.2.3 + 1.2.3`
This should saw 2 error messages!

Reply:
Added. It results in two equal messages, with different spans.